src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderDefault.java

```diff
@@ -13,21 +13,23 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.customimportorder;
 
 import static java.io.File.createTempFile;
-import static java.awt.Button.ABORT; // violation
-import static java.awt.print.Paper.*; // violation
+import static java.awt.Button.ABORT; // violation - in ASCII order 'A' should go before 'c',
+// as all uppercase come before lowercase letters
+import static java.awt.print.Paper.*; // violation - in ASCII order 'P' should go before 'c',
+// as all uppercase come before lowercase letters
 import static javax.swing.WindowConstants.*;
 
-import java.awt.Button; // violation
-import java.awt.Frame; // violation
-import java.awt.Dialog; // violation
-import java.awt.color.ColorSpace; // violation
-import java.awt.event.ActionEvent; // violation
-import javax.swing.JComponent; // violation
-import javax.swing.JTable; // violation
-import java.io.File; // violation
-import java.io.IOException; // violation
-import java.io.InputStream; // violation
-import java.io.Reader; // violation
+import java.awt.Button; // violation should be at the end
```

Comment:
Please use real violation message.

Reply:
Please review changes in similar Check https://github.com/checkstyle/checkstyle/pull/12772/files